IceCream.podspec

```diff
@@ -20,7 +20,7 @@ Pod::Spec.new do |s|
 
   s.social_media_url = 'https://twitter.com/caiyue5'
 
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '14.0'
```

Comment:
bumped the example project min deployment to 15 but repos like Pigment's Podfile are still on iOS 14 min.  so i kept this one at 14 for now